src/ert/analysis/_es_update.py

```diff
@@ -267,7 +264,7 @@ def _create_temporary_parameter_storage(
             t_surface += time.perf_counter() - t
         elif isinstance(config_node, Field):
             t = time.perf_counter()
-            matrix = source_fs.load_field(key, iens_active_index)
+            matrix = source_fs.load_parameters(key, iens_active_index)
```

Comment:
Sould be possible to join this with `SurfaceConfig` a few lines above

Reply:
Not sure how to deal with `t_surface` and `t_field` though.